package.json

```diff
@@ -8,19 +8,20 @@
   },
   "bugs": "https://github.com/maximelaboisson/spike-snipcart/issues",
   "dependencies": {
-    "reshape-standard": "^1.0.0",
-    "spike": "^1.3.0",
-    "spike-css-standards": "^1.0.0",
-    "spike-js-standards": "^2.0.1",
+    "reshape-standard": "^3.0.0",
+    "spike": "^2.2.0",
+    "spike-css-standards": "^3.0.0",
+    "spike-js-standards": "^2.0.2",
+    "spike-optimize": "^0.1.2",
     "spike-page-id": "^1.0.0",
-    "webpack": "^2.3.3"
+    "sugarml": "^0.6.0",
+    "sugarss": "^1.0.0"
```

Comment:
Bunch of version upgrades here! We move fast with this project and are always trying to keep things improving.